roaring/roaring_test.go

```diff
@@ -1457,64 +1457,87 @@ const (
 	MaxContainerVal = 0xffff
 )
 
-func BenchmarkContainerLinear(b *testing.B) {
+var bmFuncs = []func(a ...uint64) *roaring.Bitmap{roaring.NewBitmap, roaring.NewBTreeBitmap}
+var bmFuncNames = []string{"slice", "btree"}
 
-	for n := 0; n < b.N; n++ {
-		bm := roaring.NewFileBitmap()
-		for row := uint64(1); row < NumRows; row++ {
-			for col := uint64(1); col < NumColums; col++ {
-				bm.Add(row*pilosa.ShardWidth + (col * MaxContainerVal))
+func BenchmarkContainerLinear(b *testing.B) {
+	for i, bmMaker := range bmFuncs {
+		b.Run(fmt.Sprintf("%v", bmFuncNames[i]), func(b *testing.B) {
+			for n := 0; n < b.N; n++ {
+				bm := bmMaker()
+				for row := uint64(1); row < NumRows; row++ {
+					for col := uint64(1); col < NumColums; col++ {
+						bm.Add(row*pilosa.ShardWidth + (col * MaxContainerVal))
+					}
+				}
 			}
-		}
+
+		})
 	}
 }
 
 func BenchmarkContainerReverse(b *testing.B) {
-	for n := 0; n < b.N; n++ {
-		bm := roaring.NewFileBitmap()
-		for row := NumRows - 1; row >= 1; row-- {
-			for col := NumColums - 1; col >= 1; col-- {
-				bm.Add(row*pilosa.ShardWidth + (col * MaxContainerVal))
+	for i, bmMaker := range bmFuncs {
+		b.Run(fmt.Sprintf("%v", bmFuncNames[i]), func(b *testing.B) {
```

Comment:
`fmt.Sprintf("%v", bmFuncNames[i])` => `bmFuncNames[i]`

Reply:
yep... unnecessary sprintfs are not our friend